appIcons.js

```diff
@@ -513,6 +514,61 @@ const MyAppIcon = new Lang.Class({
         Clutter.cairo_set_source_color(cr, bodyColor);
         cr.fill();
         cr.$dispose();
+    },
+
+    _numberOverlay: function() {
+        // Add label for a Hot-Key visual aid
+        this._numberOverlayLabel = new St.Label();
+        this._numberOverlayBin = new St.Bin({child: this._numberOverlayLabel });
+        this._numberOverlayStyle = 'background-color: rgba(0,0,0,0.8);'
+        this._numberOverlayOrder = -1;
+        this._numberOverlayBin.hide();
+
+        // iconBox is a Shell.GenericContainer from IconGrid.BaseIcon
+        this.iconBox = new Shell.GenericContainer();
```

Comment:
iconBox as a variable name is a bit confusing I think. Also, why is iconBox "public"?